lib/namae/parser.y

```diff
@@ -3,7 +3,7 @@
 
 class Namae::Parser
 
-token COMMA UWORD LWORD PWORD NICK AND APPELLATION TITLE SUFFIX
+token COMMA UWORD LWORD PWORD NICK AND APPELLATION TITLE SUFFIX UPWORD
```

Comment:
Let's call this `UPARTICLE` for clarity -- otherwise it's hard to tell what distinguishes this from `UWORD`.